lib/coercion.ex

```diff
@@ -80,6 +85,19 @@ defmodule Coercion do
   end
   def coerce(_value, :string), do: {:invalid, ""}
 
+  # Atom
+  def coerce(value, :atom) when is_bitstring(value) do
+    atom_value =
+      value
+      |> String.trim
+      |> String.to_existing_atom
+    {:ok, atom_value}
+  end
+  def coerce(value, :atom) when is_bitstring(value), do: coerce(value, :atom)
```

Comment:
This signature looks like a duplicate of the one above.

Reply:
Done